packages/web/.env.example

```diff
@@ -6,9 +6,11 @@ NEXT_PUBLIC_PACKAGE_POSITION_PATH="gno.land/r/demo/position"
 NEXT_PUBLIC_PACKAGE_STAKER_PATH="gno.land/r/demo/staker"
 NEXT_PUBLIC_PACKAGE_NFT_PATH="gno.land/r/demo/gnft"
 NEXT_PUBLIC_GOVERNANCE_PATH="gno.land/r/demo/gov"
-NEXT_PUBLIC_WRAPPED_GNOT_PATH="gno.land/r/demo/wugnot"
 NEXT_PUBLIC_PACKAGE_ROUTER_ADDRESS="g1pjtpgjpsn4hjfv2n4mpz8cczdn32jkpsqwxuav"
 NEXT_PUBLIC_PACKAGE_POOL_ADDRESS="g15z32w7txv6lw259xzhzzmwtwmcjjc0m6dqzh6f"
 NEXT_PUBLIC_PACKAGE_POSITION_ADDRESS="g10wwa53xgu4397kvzz7akxar9370zjdpwux5th9"
 NEXT_PUBLIC_PACKAGE_STAKER_ADDRESS="g1puv9dz470prjshjm9qyg25dyfvrgph2kvjph68"
-NEXT_PUBLIC_PACKAGE_GOVERNANCE_ADDRESS="g1kmat25auuqf0h5qvd4q7s707r8let5sky4tr76"
\ No newline at end of file
+NEXT_PUBLIC_PACKAGE_GOVERNANCE_ADDRESS="g1kmat25auuqf0h5qvd4q7s707r8let5sky4tr76"
+NEXT_PUBLIC_WRAPPED_GNOT_PATH="gno.land/r/demo/wugnot"
+NEXT_PUBLIC_GNOS_TOKEN_PATH="gno.land/r/demo/gnos"
```

Comment:
GNOS -> GNS

Reply:
Thanks for letting us know what we missed.